src/aap_eda/settings/default.py

```diff
@@ -195,14 +195,23 @@ def _get_boolean(name: str, default=False) -> bool:
     "ansible_base.resource_registry",
     "ansible_base.jwt_consumer",
     "ansible_base.rest_filters",
+    "ansible_base.feature_flags",
     # Local apps
     "aap_eda.api",
     "aap_eda.core",
 ]
 
+
+def toggle_feature_flags(_flags, _settings):
```

Comment:
Just a tickle request, can you just remove the underscore for `_flags` and `_settings_` variables? the underscore is used to indicate a private context which I think has no meaning here.

Reply:
@Alex-Izquierdo Done 👍

Reply:
thanks and sorry :)